config/constants.go

```diff
@@ -122,6 +123,10 @@ const (
 	// the init containers and containers for a driver or worker pod.
 	WorkspaceVolumeName = "workspace"
 
+	// WorkerContainerName holds the name of the worker container where the test is
```

Comment:
Are we still using this constant?

Reply:
I thought I have delete it, let me fix it.